Lib/fontTools/varLib/__init__.py

```diff
@@ -739,15 +764,28 @@ def load_designspace(designspace):
 	for obj in masters+instances:
 		obj_name = obj.name or obj.styleName or ''
 		loc = obj.location
+		if loc is None:
+			raise VarLibValidationError(
+				f"Source or instance '{obj_name}' has no location."
```

Comment:
when can happen that an object.location is None? If anything it should be an empty dict..
Do we not fill the implicit locations with axes defaults?

Reply:
Happens when you have no `<location>` element in a DS.

Reply:
Note to self: maybe use `designspaceLib.tagForAxisName` for what `standard_axis_map` does.